### DISCUSSION TITLE
Zigbee2MQTT: Default installation asking for login credentials (ssh)

### DISCUSSION THREAD

**Discussion by sebwar2004** [tteck/Proxmox#1054] [Help Needed] [answered]
Hi,
first: thank you very much for your awesome scripts!

Unfortunately I need some help with the Zigbee2MQTT LXC:
I installed it with default settings and wanted to edit the configuration.yaml afterwards, but within the console it's asking for login credentials - and I really don't know what to use here :-(

Hope you have a solution for me, unfortunately I can't reinstall the LXC at the moment because my home/server internet doesn't work at the moment...

Thanks again and best regards from Germany,
Seb

**Comment by tteck** [comment, marked as answer]
You can not SSH in without a root password

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/385

**Comment by bobendylon** [comment]
Dear all,
i can't connect to my console Zigbee2mqtt.
It asks login ? What it is? and password?
![image](https://github.com/tteck/Proxmox/assets/93086987/2b7dfcf9-6727-48d1-a172-760f389dbdbd)

**Reply by tteck** [reply]
❗ It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.

**Comment by ThiasJ** [comment]
Hi @tteck 
I had the same "problem". I tried both, the debian and alpine version, but in both cases I got prompted with a login.

In the end it was my fault, because I didnt see the question:
![image](https://github.com/user-attachments/assets/5d78d71b-700e-440d-89c5-bbaf7be9e085)
In the first two tries I closed the window because I didnt watch closely and thougt the install is done.

After answering the question I can login. Maybe it would make sense if you update the https://tteck.github.io/Proxmox/#zigbee2mqtt-lxc page to look out for this question?
I think @sebwar2004 and @bobendylon ran into the same mistake?

**Reply by tteck** [reply]
> Maybe it would make sense if you update the https://tteck.github.io/Proxmox/#zigbee2mqtt-lxc page to look out for this question?

![Screenshot 2024-07-30 2 43 16 PM](https://github.com/user-attachments/assets/99c695fd-2d06-4f8b-a150-8d123496f5d7)

**Reply by tteck** [reply]
> alpine version, but in both cases I got prompted with a login.

With Alpine, you have to type `root`
